lib/services/azuresqldb/cmd-provision.js

```diff
@@ -166,8 +166,12 @@ var sqldbProvision = function (log, params) {
                     } else if (firewallRules !== null) {
                         var errorMessage = util.format('You specify the firewall rules for the server but you do not have the privilege to operate the server.');
                         callback(Error(errorMessage));
-                    } else {
+                    } else if (result.statusCode === HttpStatus.OK) {
+                        fullyQualifiedDomainName = JSON.parse(result.body).properties.fullyQualifiedDomainName;
                         callback(null);
+                    } else {
+                        var errorMessage = util.format('Unexpected error: %j', result);
+                        callback(Error(errorMessage));
                     }
                 },
                 function (callback) {   // see if db exists (in the case that sql server was there already)
```

Comment:
@zhongyi-zhang The integration test case is missing for the existing sql server.